frontend/lib/location/dialogs.dart

```diff
@@ -46,20 +24,22 @@ class LocationPermissionDialog extends StatelessWidget {
 class RationaleDialog extends StatelessWidget {
   final String _rationale;
 
-  const RationaleDialog({Key? key, required String rationale}) : _rationale = rationale, super(key: key);
+  const RationaleDialog({Key? key, required String rationale})
+      : _rationale = rationale,
+        super(key: key);
 
   @override
   Widget build(BuildContext context) {
     return AlertDialog(
       title: const Text("Standortberechtigung"),
-      content:Column(
+      content: Column(
         crossAxisAlignment: CrossAxisAlignment.start,
         mainAxisSize: MainAxisSize.min,
         children: <Widget>[
           Text(_rationale),
-          const Text('Soll nochmal nach der Berechtigung gefragt werden?')
+          const Text('Soll nocheinmal nach der Berechtigung gefragt werden?')
```

Comment:
Correct would be `noch einmal` or `nochmals`: https://www.duden.de/rechtschreibung/nochmals